test/test_pipeline/components/feature_preprocessing/test_extra_trees_regression.py

```diff
@@ -35,7 +35,7 @@ def test_default_configuration_regression(self):
             predictor = regressor.fit(X_train_trans, Y_train)
             predictions = predictor.predict(X_test_trans)
             accuracy = sklearn.metrics.mean_squared_error(predictions, Y_test)
-            self.assertAlmostEqual(accuracy, 20.193400000000004, places=2)
+            self.assertAlmostEqual(accuracy, 17.93480710955056, places=2)
```

Comment:
The accuracies decreased. Couldn't figure out why, even after carefully inspecting the list of changes.

Reply:
The metric is actually misnamed. Could you please change the metric to `error`? Then it's clear that it actually improved by going to 0.22.2.

Reply:
Ah! This is good news. :-)
I'll rename it.